Readme - second attempt to improve it

matkoniecz opened the pull request:
I attempted to break sentences into smaller, moved things that matter to OSM power users to the end or in case of Osmose removed it, added an example, rephrased one thing as it was technically untrue etc

-----

My work on this pull request and UX testing was sponsored by a NGI Zero Discovery [grant](https://www.openstreetmap.org/user/Mateusz%20Konieczny/diary/368849)

westnordost commented:
You are missing some articles.

Also (as the current readme?), the text speaks of issues but makes no mention where they come from. I'd rather not call them issues but something along the lines of places where information can be added. So, in turn, the text should mention that the app searches for these places and presents them to the user as markers on the map.

matkoniecz commented:
- added articles (pinging @goldfndr for additional review and checking is it more or less clear)
- added info that StreetComplete finds places to survey on its own
- move basic info to the first paragraph, explain process with example in the second
 mention screeeshots section for people who prefer visual info, rather than text description

westnordost commented on README.md:
```diff
@@ -1,16 +1,15 @@
 
-StreetComplete is an easy to use editor of OpenStreetMap data available for Android. It can be used without any OpenStreetMap-specific knowledge. It asks simple questions, with answers directly used to edit and improve OpenStreetMap data.
+StreetComplete is an easy to use editor of OpenStreetMap data available for Android. It can be used without any OpenStreetMap-specific knowledge. It asks simple questions, with answers directly used to edit and improve OpenStreetMap data. The app is aimed at users who do not know anything about OSM tagging schemes but still want to
+contribute to the OpenStreetMap by surveying their neighbourhood (or other places as well).
```
👍

westnordost commented on README.md:
```diff
@@ -1,16 +1,15 @@
-of the user's OSM account.
+ StreetComplete on its own locates places where survey will be helpful. The found places are shown as markers on a map. Each can be solved by walking to the location and filling in the app a simple form to complete/correct the information. For example a road without name will result in a marker with an appropriate quest. After tapping it "What is the name of this road?" question is shown, with a  ...
+
+The user's answer is automatically processed and directly uploaded into the OSM database. Edits are done in meaningful changesets using user's OSM account.
```
```suggestion
The user's answer is automatically processed and directly uploaded into the OSM database. Edits are done in meaningful changesets using the user's OSM account.
```

westnordost commented on README.md:
```diff
@@ -1,16 +1,15 @@
 
-The app is aimed at users who do not know anything about OSM tagging schemes but still want to
-contribute to the OpenStreetMap by surveying their neighbourhood (or other places as well). Because of the target group, only those quests are shown which are answerable very clearly by asking one simple question and only those quests are created which contain very few false positives.
+To make app easy to use quests are limited to answerable by asking simple questions. Also, only quests with very few false positives are allowed. For example street name quest is handling also roads without real names - both by skipping roads with `noname=yes` tag set and by allowing user to specify that road is really unnamed.
```
```suggestion
To make the app easy to use, quests are limited to those answerable by asking simple questions. 
```

> Also, only quests with very few false positives are allowed. For example street name quest is handling also roads without real names - both by skipping roads with `noname=yes` tag set and by allowing user to specify that road is really unnamed.

Would need a more thorough explanation what is meant by that. The second sentence is confusing. Explaining it properly would be too much for the README.

westnordost commented on README.md:
```diff
@@ -1,16 +1,15 @@
-solvable by filling out a simple form to complete/correct the information on site. The user's
-answer is then processed and directly uploaded into the OSM database in meaningful changesets in the name
-of the user's OSM account.
+ StreetComplete on its own locates places where survey will be helpful. The found places are shown as markers on a map. Each can be solved by walking to the location and filling in the app a simple form to complete/correct the information. For example a road without name will result in a marker with an appropriate quest. After tapping it "What is the name of this road?" question is shown, with a  ...
```
```suggestion
 StreetComplete automatically looks for places where a survey is needed and shows those as quest markers on the map. Each of these quests can then be solved on site by answering one simple question. For example, tapping on a marker may show the question "What is the name of this road?", with a text field to answer it. 
More examples are shown in screenshots below.
```

westnordost commented:
👍
